groovy/inductor_dashboard.groovy

```diff
@@ -0,0 +1,376 @@
+                        docker system prune -f
+                        docker login ccr-registry.caas.intel.com
+                        docker pull ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag}
+                    fi
+                '''
+            }           
+        }else {
+            sh '''
+            #!/usr/bin/env bash
+            tag=${image_tag}
+            old_container=`docker ps |grep pt_inductor |awk '{print $1}'`
+            if [ -n "${old_container}" ]; then
+                docker stop $old_container
+                docker rm $old_container
+                docker container prune -f
+            fi
+            old_image_id=`docker images|grep pt_inductor|grep ${tag}|awk '{print $3}'`
+            old_image=`echo $old_image_id| awk '{print $1}'`
+            if [ -n "${old_image}" ]; then
+                docker rmi -f $old_image
+            fi
+            docker system prune -f
+            mv docker/Dockerfile ./
+            DOCKER_BUILDKIT=1 docker build --no-cache --build-arg http_proxy=${http_proxy} --build-arg https_proxy=${https_proxy} --build-arg BASE_IMAGE=${BASE_IMAGE}--build-arg PT_REPO=${PT_REPO} --build-arg PT_BRANCH=${PT_BRANCH} --build-arg PT_COMMIT=${PT_COMMIT} --build-arg TORCH_VISION_BRANCH=${TORCH_VISION_BRANCH} --build-arg TORCH_VISION_COMMIT=${TORCH_VISION_COMMIT} --build-arg TORCH_DATA_BRANCH=${TORCH_DATA_BRANCH} --build-arg TORCH_DATA_COMMIT=${TORCH_DATA_COMMIT} --build-arg TORCH_TEXT_BRANCH=${TORCH_TEXT_BRANCH} --build-arg TORCH_TEXT_COMMIT=${TORCH_TEXT_COMMIT} --build-arg TORCH_AUDIO_BRANCH=${TORCH_AUDIO_BRANCH} --build-arg TORCH_AUDIO_COMMIT=${TORCH_AUDIO_COMMIT} --build-arg TORCH_BENCH_BRANCH=${TORCH_BENCH_BRANCH} --build-arg TORCH_BENCH_COMMIT=${TORCH_BENCH_COMMIT} --build-arg BENCH_COMMIT=${BENCH_COMMIT} -t ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag} -f Dockerfile --target image .
+            '''
+        } 
+    }
+    stage('login container & prepare scripts & run') {
+        echo 'login container & prepare scripts & run......'
+        if ("${isOP}" == "true") {
+            sh '''
+            #!/usr/bin/env bash
+            tag=${image_tag}
+            docker run -tid --name op_pt_inductor --privileged --env https_proxy=${https_proxy} --env http_proxy=${http_proxy} --net host  --shm-size 1G -v ${WORKSPACE}/opbench_log:/workspace/pytorch/dynamo_opbench ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag}
+            docker cp scripts/microbench/microbench_parser.py op_pt_inductor:/workspace/pytorch
+            docker cp scripts/microbench/microbench.sh op_pt_inductor:/workspace/pytorch
+            docker exec -i op_pt_inductor bash -c "bash microbench.sh dynamo_opbench ${op_suite} ${op_repeats} ${DT};cp microbench_parser.py dynamo_opbench;cd dynamo_opbench;pip install openpyxl;python microbench_parser.py -o ${_VERSION} -l ${BUILD_URL} -n ${_NODE};rm microbench_parser.py"
+            '''
+        }else {

```

Comment:
Change the `else` to `else if ("${ModelBench}" == "true")`  and also add `else if ("{LLMBench}" == "true")` for `GPT-J` and others LLMs test

Reply:
Do we also use this job for `LLMBench` ?  Maybe we need to install `transformers`explicitly . Current dockerfile  with version 4.26.1, which failed to run `TypeError: 'int' object is not iterable` https://github.com/huggingface/transformers/blob/v4.26.1/src/transformers/generation/utils.py#L2229

Reply:
`conda install -y -c huggingface transformers` run success.

Reply:
can we use `pip install transformers==xxx` to solve it?

Reply:
`pip install transformers==4.24.0` work.

Reply:
Let's use `pip install` to instead of `conda install`